Firestore/core/src/firebase/firestore/local/CMakeLists.txt

```diff
@@ -77,6 +77,11 @@ cc_library(
 cc_library(
   firebase_firestore_local
   SOURCES
+    document_key_reference.cc
+    document_key_reference.h
+    index_free_query_engine.h
```

Comment:
Please sort contents (.cc should be before .h)

Reply:
Done